modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2631,17 +2631,28 @@ export class Wallet implements IWallet {
   private async prebuildTransactionTss(params: PrebuildTransactionOptions = {}): Promise<PrebuildTransactionResult> {
     const reqId = params.reqId || new RequestTracer();
     this.bitgo.setRequestTracer(reqId);
+
+    if (
+      _.isFunction(params.customCommitmentGeneratingFunction) &&
+      _.isFunction(params.customGShareGeneratingFunction) &&
+      _.isFunction(params.customRShareGeneratingFunction)
+    ) {
+      // force txRequest full API version for custom signing
+      params.apiVersion = 'full';
+    }
+
```

Comment:
Should we fail the call with a error message similar to below ("using custom signing must use 'full' api version")?

Reply:
The use case for external signing has always been using full version, https://github.com/BitGo/BitGoJS/blob/e15f9b8d09e50649121f5a5c6bf5a8f4bad0b283/modules/sdk-core/src/bitgo/utils/tss/eddsa/eddsa.ts#L480-L524.

Reply:
So instead of adapting the signing process to keep splitting to lite vs full, i rather force all to full.